admin/endpoints/class-endpoint-indexable.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Endpoints
+ */
+
+/**
+ * Represents an implementation of the WPSEO_Endpoint interface to register one or multiple endpoints.
+ */
+class WPSEO_Endpoint_Indexable implements WPSEO_Endpoint, WPSEO_Endpoint_Storable {
+
+	const REST_NAMESPACE = 'yoast/v1';
+	const ENDPOINT_RETRIEVE = 'indexables/(?P<object_type>.*)/(?P<object_id>\d+)';
```

Comment:
Is it planned to only use this endpoint for read operations? Otherwise I'd consider renaming this constant to something like `ENDPOINT_SINGULAR`, as it's not only for retrieving an item. You could theoretically also `POST` or `PUT` to that endpoint to modify the indexable.

This would make it safe for the future where there could be full CRUD support (and there could be an `ENDPOINT_PLURAL` for the more general `indexables/(?P<object_type>.*)` (or maybe even just `indexables` without object type).

Reply:
@andizer I see you changed `indexables` to `indexable`, was that decided in another discussion? I think `indexables` is fine, the `SINGULAR` in the name just represents the presence of the `object_id` part in the endpoint, making it an endpoint for a singular item.